pkg/console/starter/starter.go

```diff
@@ -245,13 +257,14 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		operatorClient,
 		kubeClient.CoreV1(),
 		configInformers.Config().V1().Authentications(),
+		configInformers.Config().V1().FeatureGates(),
 		configClient.ConfigV1().Authentications(),
 		operatorConfigInformers.Operator().V1().Consoles(),
-		apiextensionsInformers.Apiextensions().V1().CustomResourceDefinitions(),
 		kubeInformersConfigNamespaced.Core().V1().ConfigMaps(),
 		kubeInformersNamespaced.Core().V1().Secrets(),
 		kubeInformersNamespaced.Core().V1().ConfigMaps(),
 		kubeInformersNamespaced.Apps().V1().Deployments(),
+		featureGateAccessor,
```

Comment:
pass a gate value here.  like this: https://github.com/openshift/enhancements/blob/06fce4b637487078bc9834378cb8d4a4cb055ae6/dev-guide/featuresets.md?plain=1#L190-L195